packages/components/MenuButton/src/MenuButton.types.ts

```diff
@@ -26,11 +27,13 @@ export interface MenuButtonProps extends IButtonProps {
   menuItems?: MenuButtonItemProps[];
   onItemClick?: (key: string) => void;
   contextualMenu?: ContextualMenuProps;
+  primary?: boolean;
 }
 
 export type MenuButtonSlotProps = {
   root: MenuButtonProps;
   button: IButtonProps & MenuButtonTokens;
+  primaryButton: IButtonProps & MenuButtonTokens;
```

Comment:
What is the `primaryButton` slot used for relative to the `button` slot?  At a glance this feels like it could be handled w/ just `button` and that we shouldn't need multiple, but I'm plausibly missing important context or purpose :)

Reply:
PrimaryButton is a different component that's why I used a separate slot for this

Reply:
I think Pat's right here, we shouldn't introduce a new slot just to change it to a primary button.  They take the same props & tokens and only one of the two is ever rendered.

If we were using the vNext Button instead of the one we're deprecating we could simply put primary: true on the slotProps

Reply:
I agree but in this case I'll need to do redesign for the experimental Button. We agreed in the chat on the beginning that I'll work on Button component from components folder. I'm not sure if I can use one slot for two different components.